### PR TITLE
[Project Submission] Pristorage

### PR DESCRIPTION
**Pull request by manhnvan**

### CODE REVIEW

**Comment by manhnvan** [comment]
@jennijuju please take a review and add Pristorage updates to Filecoin Newsletter. Thank you so much!

File: projects/application/Pristorage.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+|      Name      |          Email          |              Github              |          Role          |
+|:--------------:|:-----------------------:|:--------------------------------:|:----------------------:|
+| Nguyen Van Manh|    manhnv@giong.network |     https://github.com/manhnvan  |  Primary Developer     |
+| Tuan Pham Minh |    tuanpm@giong.network |     https://github.com/iamvon    | Mentor                 |
+
+# How the community can engage
+GitHub Discussion: https://github.com/filecoin-project/community/discussions/358
+Email: manhnv@giong.network
+Telegram: manhnvm
```
Comment (longfeiWan9) [inline projects/application/Pristorage.md:59]:
```suggestion
+ GitHub Discussion: https://github.com/filecoin-project/community/discussions/358
+ Email: manhnv@giong.network
+ Telegram: manhnvm
```

**Comment by longfeiWan9** [comment]
@manhnvan I add a small suggestion, please take a look.

**Comment by manhnvan** [comment]
> @manhnvan I add a small suggestion, please take a look.
thanks you for your suggestion, hope you interested in my application

**Pull request merged**